cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PerformanceBenchmark.scala

```diff
@@ -93,6 +93,10 @@ object PerformanceBenchmark extends GzipGraphDownloader {
   val getNodeFlag = flags("gn", 0, "run getNodeById benchmark with a given number of steps")
   val reps = flags("reps", DEFAULT_REPS, "number of times to run benchmark")
   val adjacencyList = flags("a", false, "graph in adjacency list format")
+  val gzipFlag = flags("gz", false, "Is file in gzip format")
+  val dirOutFlag = flags("out", false, "Graph direction: out")
```

Comment:
I think you want the default values to be true for both dirOutFlag and dirInFlag so that graphDir = BothInOut by default